portfolio/src/main/webapp/script.js

```diff
@@ -50,3 +51,130 @@ function appendCommentElement(txt, commentDiv) {
+        elementType: 'geometry',
+        stylers: [{color: '#263c3f'}]
+      },
+      {
+        featureType: 'poi.park',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#6b9a76'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'geometry',
+        stylers: [{color: '#38414e'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'geometry.stroke',
+        stylers: [{color: '#212a37'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#9ca5b3'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'geometry',
+        stylers: [{color: '#746855'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'geometry.stroke',
+        stylers: [{color: '#1f2835'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#f3d19c'}]
+      },
+      {
+        featureType: 'transit',
+        elementType: 'geometry',
+        stylers: [{color: '#2f3948'}]
+      },
+      {
+        featureType: 'transit.station',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#d59563'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'geometry',
+        stylers: [{color: '#17263c'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#515c6d'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'labels.text.stroke',
+        stylers: [{color: '#17263c'}]
+      }
+    ]
+  });
+
+  const kcMarker = new google.maps.Marker({
+    position: {lat: 39.099724, lng: -94.578331},
+    map: map,
+    animation: google.maps.Animation.DROP,
+    title: 'Kansas City, Kansas'
+  });
+  const culvMarker = new google.maps.Marker({
+    position: {lat: 41.2224, lng: -86.4130},
+    map: map,
+    animation: google.maps.Animation.DROP,
+    title: 'Culver, Indiana'
+  });
+  const nashMarker = new google.maps.Marker({
+    position: {lat: 36.1627, lng: -86.7816},
+    map: map,
+    animation: google.maps.Animation.DROP,
+    title: 'Nashville, Tennessee'
+  });
+
+  const kcString = '<h4>Kansas City, Kansas</h4>' +
+      '<p>My hometown</p>';
+  const culvString = '<h4>Culver, Indiana</h4>' +
+      '<p>Where I attended a boarding high school for four years</p>';
+  const nashString = '<h4>Nashville, Tennessee</h4>' +
+      '<p>Where I am currently attending college</p>';
+
+  var kcInfoWindow = new google.maps.InfoWindow({content: kcString});
+  var culvInfoWindow = new google.maps.InfoWindow({content: culvString});
+  var nashInfoWindow = new google.maps.InfoWindow({content: nashString});
+
+  setMarker(kcMarker, kcInfoWindow);
+  setMarker(culvMarker, culvInfoWindow);
+  setMarker(nashMarker, nashInfoWindow);
+}
+
+function setMarker(marker, infoWindow) {
+  marker.setMap;
```

Comment:
What is this line doing?  setMap is a method, so it should take some arguments, right?  Should it be marker.setMap(map)?  The map seems to be working when I interact with it, so I'm not sure what the deal is... maybe this line is unnecessary?

Reply:
Oh also I see when you're creating the markers using "new google.maps.Marker(..." above, one of the fields you are passing in is "map: map", so I think the markers already have their map value set.  So I think that this line here can likely be deleted =)